bson/src/main/org/bson/RawBsonDocument.java

```diff
@@ -244,6 +245,21 @@ public int size() {
     }
 
     @Override
+    public String getFirstKey() {
```

Comment:
And might as well optimize it here rather than force creation of a keySet.

Reply:
Ack